tensorflow/python/keras/distribute/distributed_training_utils.py

```diff
@@ -645,6 +645,8 @@ def _prepare_feed_values(model, inputs, targets, sample_weights, mode):
     targets = []
   elif not is_distributing_by_cloning(model):
     sample_weights = None  # b/129503665
+  elif sample_weights is not None:
+    sample_weights = training_utils.standardize_sample_weights(sample_weights, model.outputs)
   else:
     sample_weights = [
```

Comment:
Should sample_weights just be equal to None here? I think with the latest change we don't need to set sample_weights to an array of Nones.